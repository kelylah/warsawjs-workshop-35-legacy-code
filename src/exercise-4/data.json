[
  {
    "type": "sales",
    "data": [
      { "date": "2019-06-28", "sell": 2400 },
      { "date": "2019-07-31", "sell": 1200 },
      { "date": "2019-08-30", "sell": 1400 }
    ]
  },
  {
    "type": "targets",
    "data": [
      { "date": "2019-06-01", "target": 2200 },
      { "date": "2019-07-01", "target": 2000 },
      { "date": "2019-08-01", "target": 2100 }
    ]
  }
]
