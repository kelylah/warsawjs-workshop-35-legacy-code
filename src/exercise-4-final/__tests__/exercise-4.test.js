import app from '../exercise-4.js';
import data from '../data.json';

test('', () => {
  expect(app(data)).toMatchSnapshot();
});
