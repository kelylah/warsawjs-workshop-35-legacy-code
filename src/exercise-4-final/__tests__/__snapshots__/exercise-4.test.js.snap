// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
Array [
  Object {
    "date": "2019-08-31",
    "sell": 1400,
    "target": 2100,
  },
  Object {
    "date": "2019-07-31",
    "sell": 1200,
    "target": 2000,
  },
  Object {
    "date": "2019-06-30",
    "sell": 2400,
    "target": 2200,
  },
]
`;
